connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java

```diff
@@ -780,6 +774,14 @@ protected void stopServices() {
         }
     }
 
+    // Timeout for herderExecutor to gracefully terminate is set to a value to accommodate
+    // reading to the end of the config topic + successfully attempting to stop all connectors and tasks and a buffer of 10s
+    private long getHerderExecutorTimeoutMs() {
```

Comment:
Nit: We don't use `get` in this code base

```suggestion
    private long herderExecutorTimeoutMs() {
```

Also, is there a reason this has to be a separate method at all? It's only called in one place right now and the logic isn't really complicated enough to warrant isolated unit testing.

Reply:
I created a separate method for this as it appeared slightly cleaner and in the future if we want to add more parameters when calculating the herderExecutor, the change would be made at one place. Don't have a strong preference on this and if you feel I should remove it, can do that.